src/app/post-page/post-page.component.ts

```diff
@@ -17,9 +21,48 @@ export class PostPageComponent implements OnInit {
    * Users should be able to navigate between the posts as well, i.e. to the previous or next post.
    */
 
-  constructor() { }
+  post: any; // izit any?
+  comments: any; // izit any?
+
+  constructor(
+    private _postService: PostService,
+    private _commentService: CommentService,
+    private _router: Router,
+  ) { }
 
   ngOnInit(): void {
+    this.post = this._postService.getPostById(this.post.id); // get post id from where?
+    this.comments = this._commentService.getCommentsByPost(this.post.id);
```

Comment:
same problem as above. do note that to minimize resources, the data sent from the backend **will be paginated**. thus, u need to handle navigation within the comments as well.